src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrManagerBase.cs

```diff
@@ -271,12 +278,17 @@ protected async Task<IEnumerable<VmrDependencyUpdate>> GetAllDependenciesAsync(
                         $"for a {VersionFiles.VersionDetailsXml} dependency of {dependency.Name}");
                 }
 
+                var build = (await barClient.GetBuildsAsync(dependency.RepoUri, dependency.RepoUri))
```

Comment:
Should the arguments be uri + sha or something?

Reply:
yes.. fixed